iot/api/CloudIotTest/CloudIotTest.cs

```diff
@@ -0,0 +1,334 @@
+﻿// Copyright(c) 2018 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+using Google.Cloud.Iam.V1;
+using Google.Cloud.PubSub.V1;
+using Grpc.Core;
+using Xunit;
+using System;
+
+namespace GoogleCloudSamples
+{
+    // <summary>
+    /// Runs the sample app's methods and tests the outputs
+    // </summary>
+    public class CommonTests
+    {
+        private static readonly string s_projectId = Environment.GetEnvironmentVariable("GOOGLE_PROJECT_ID");
+        private static readonly string s_regionId = "us-central1";

```

Comment:
Should this be specified by customer to work for them?
Similar comment for line 30.
If yes, please, replace the value with something like "SPECIFY_GCP_REGION_HERE"...

Reply:
If this shouldn't be specified by the customer, please, add a comment to explain why it's always us-cental1.

Reply:
Nope, this is just for testing, using the hardcoded values should be totally acceptable.